guides/manual_release.md

```diff
@@ -169,7 +169,8 @@ Example:
    `[RELEASE_BRANCH_VERSION-]` with the release branch number (e.g., `1.4`)
 
    ```
-   cd config/prow
+   # run gcloud auth login first and pull the latest commit in main branch
```

Comment:
can this be scriptified? ie. shouldn't we just call `gcloud auth login` before here?

Reply:
No, because gcloud auth login is a browser based flow. You are already expected to run that for access to the clusters and other things.